PodTemplates.yaml

```diff
@@ -60,7 +59,7 @@ spec:
     env:
     - name: "HOME"
       value: "/home/helm"
-    image: "olblak/updatecli:v0.0.10"
+    image: "v0.0.13"
```

Comment:
This doesn’t look right

Reply:
You are right, I just fixed the configuration [here](https://github.com/jenkins-infra/charts/pull/306/commits/31d9e2f22abeb0ca2ac4ccdf4b742d0335c46f91)